stock_account_move_reset_to_draft/readme/USAGE.rst

```diff
@@ -1,7 +1,7 @@
-#. Create a product category with Costing Method: Average Cost (AVCO)
+#. Create a product category with Costing Method: Average Cost (AVCO) or First In First Out (FIFO).
 #. Create a product linked to the category created before.
 #. Create a purchase order and adds a product line with quantity 1 and price 10.
 #. Confirms the purchase order and validates the incoming picking.
 #. Creates an invoice from the purchase order.
 #. Changes the invoice line price to 12 and confirm the invoice.
-#. It is possible to reset the invoice to draft (the SVL of the difference will be removed).
+#. It is possible to reset the invoice to draft (the record to offset the SVL difference will be created).
```

Comment:
```suggestion
#. It is possible to reset the invoice to draft (a new SVL record to offset the existing SVL difference will be created).
```

Reply:
We may also add this explanation: https://odoo-community.org/groups/contributors-15/contributors-1965466